docs/integrations/log_auditbeat_linux.md

```diff
@@ -0,0 +1,327 @@
+
+  # Detect changes to files included in subdirectories. Disabled by default.
+  recursive: false
+
+- module: system
+  datasets:
+    - package   # Installed, updated, and removed packages
+  period: 2m
+
+- module: system
+  datasets:
+    - host      # General host information, e.g. uptime, IPs
+    - login     # User logins, logouts, and system boots.
+    - process   # Started and stopped processes
+    - socket    # Opened and closed sockets
+    - user      # User information
+
+
+  user.detect_password_changes: true
+
+# ================================== Outputs ===================================
+# ---------------------------- Elasticsearch Output ----------------------------
+output.elasticsearch:
+  enabled: false
+
+# -------------------------------- File Output ---------------------------------
+output.file:
+  enabled: true
+
+  # Configure JSON encoding
+  codec.json:
+    #pretty: false
+    # Configure escaping HTML symbols in strings.
+    #escape_html: false
+
+  # Path to the directory where to save the generated files.
+  path: "/tmp/auditbeat"
+  filename: auditbeat
+  rotate_every_kb: 10000
+  number_of_files: 7
+  permissions: 0600
+
+# =================================== Paths ====================================
+
+path.home: "/usr/share/auditbeat/bin"
+path.config: "/etc/auditbeat"
+path.data: "/var/lib/auditbeat"
+path.logs: "/var/log/auditbeat"
+
+# ================================== Template ==================================
+
+# Elasticsearch template settings
+setup.template.settings:
+  # A dictionary of settings to place into the settings.index dictionary
+  index:
+    number_of_shards: 1
+    #codec: best_compression
+
+# ================================== Logging ===================================
+
+logging.level: info
+
+logging.to_files: true
+logging.files:
+  path: /var/log/auditbeat
+  name: auditbeat
+  rotateeverybytes: 10485760 # = 10MB
+  keepfiles: 7
+  permissions: 0600
+
+# ================================= Migration ==================================
+
+# This allows to enable 6.7 migration aliases
+#migration.6_to_7.enabled: false
+```
+
+In order to make it easier, you can download the auditbeat configuration file:
+
+```bash
+$ wget -O /etc/auditbeat/auditbeat.yml https://docs.sekoia.io/integrations/configurations/auditbeat.yml
+```
+
+> Don't forget to set the right level of permission of the new `auditbeat.yml` if you edited a new one.
+> `sudo chmod 0600 auditbeat.yml`
+> Plus in this case ensure the owner of the file is root: `sudo chown root:root auditbeat.yml`
+
+Auditbeat comes with predefined assets for parsing, indexing, and visualizing your data. To load these assets:
+```bash
+sudo auditbeat -e
+```
+
+> If no error occurs, use `Ctrl + C` to stop running it in the terminal.
+
+Start AuditBeat with the folling command:
+```bash
+sudo systemctl restart auditbeat.service
+```
+
+> Check your logs in `/tmp/auditbeat/auditbeat` in JSON format.
+
+#### Configure the client Rsyslog

```

Comment:
Maybe for Rsyslog we could redirect the user to the deidcated documentation page and only display here things that will be specific to auditbeat

Reply:
The documentation is oriented to non expert who has no time understanding a new technologie. This is espacially the objective of the Wiki. Help people proceed to a step by step on tailor made approach.